src/input.rs

```diff
@@ -232,15 +233,24 @@ impl Action {
             },
             Action::Command(ref program, ref args) => {
                 trace!("running command: {} {:?}", program, args);
-                match Command::new(program)
+
+                #[cfg(not(windows))]
+                let spawned = Command::new(program)
                     .args(args)
                     .before_exec(|| {
```

Comment:
Please `cfg` the `before_exec` method rather than the entire `Command::new` call.

Reply:
As far as I can tell you can't? Attributes can be applied to expressions, but `.before_exec(...)` isn't an expression.

Doing that fails for me:

```rust
error: expected one of `.`, `;`, `?`, or an operator, found `#`
   --> src\input.rs:239:21
    |
238 |                     .args(args)
    |                                - expected one of `.`, `;`, `?`, or an operator here
239 |                     #[cfg(not(windows))]
    |                     ^ unexpected token
```

Reply:
You can't put an attribute on `.before_exec(...)` because it isn't an expression, and you can't put the attribute on the lib call because `before_exec` is a UNIX only extension.

How were you thinking of doing it?

Reply:
Ahh, I didn't realize `before_exec` was only on UNIX systems. I figured we could have defined functions instead of a closure and use an empty one on Windows.

Since that's not an option, this is fine for now.